google-cloud-container/google-cloud-container.gemspec

```diff
@@ -20,7 +20,7 @@ Gem::Specification.new do |gem|
 
   gem.required_ruby_version = ">= 2.0.0"
 
-  gem.add_dependency "google-gax", "~> 1.0.1"
+  gem.add_dependency "google-gax", "~> 1.0"
```

Comment:
Thank you for this change. I don't see why we would want to pin the dependency to >= 1.0 and < 1.1, especially since google-gax is following semver.